extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -276,6 +277,17 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 	}
 
+	async addNotebookToBook(): Promise<void> {
+		const books: BookModel[] = this.books.filter(b => {
+			if (!b.isNotebook) {
```

Comment:
filter usually has you returning true or false, so it'd be preferable to do that instead